code/game/objects/items/weapons/autolathe_disk/blackshield_disks.dm

```diff
@@ -6,27 +6,28 @@
 	license = 20
 
 	designs = list(
-		/datum/design/autolathe/gun/ladon = 2,
+		/datum/design/autolathe/gun/ladon,
 		//SMGs
+		/datum/design/autolathe/gun/ppv = 3,
 		/datum/design/autolathe/gun/triage = 3,
 		//rifles
 		/datum/design/autolathe/gun/watchtower = 3,
 		/datum/design/autolathe/gun/sol = 3,
 		/datum/design/autolathe/gun/omnirifle = 4,
 		/datum/design/autolathe/gun/rds_omnicarbine = 5,
-		//shotgun
+		//shotguns
+		/datum/design/autolathe/gun/bounty_lever = 3,
 		/datum/design/autolathe/gun/hustler = 5,
 		//machinegun
-		/datum/design/autolathe/gun/heroic = 5,
+		/datum/design/autolathe/gun/heroic = 6,
 		//grenade launcher
 		/datum/design/autolathe/gun/grenade_launcher_china_lake = 4,
-		//Flaregun
-		/datum/design/autolathe/gun/flare_gun,
+		//Frames
+		/datum/design/autolathe/gun_frame/discountnordwind,
+		/datum/design/autolathe/gun_frame/omni = 2,
+		/datum/design/autolathe/gun_frame/dp = 4,
 		//Misc
 		/datum/design/autolathe/device/landmine = 0,
-		/datum/design/autolathe/sec/riot = 2,
```

Comment:
why are you removing these three
And the flare gun

Reply:
Security misc disk / vendor / teleporter is already where you get these, doesn't line up with the disk theme of being BLACKSHIELD made equipment, shields are more marshals